docs/client/initializing.md

```diff
@@ -0,0 +1,81 @@
+# <img align="center" src="../images/logo.png">  Initializing Your Java Client
+
+The first step to using your generated client in code is to import and initialize your client. Our SDKs are modelled such
+that the client is the main point of access to the generated code.
+
+## Importing Your Client
+
+You include the package from the namespace specified when generating (under flag `--namespace`). For the sake of this example,
```

Comment:
We don't actually require importing the client manually in the code so I would use this section to specify how to import the maven dependency.

Reply:
Something like:

You include the generated client through Maven. In your application's `pom.xml` add a dependency with `groupId`, `artifactId`, and `version` specified in the generated client's `pom.xml` file. For the sake for this example, let's say the `groupId` is the same as the package name specified when generating (under flag `--namespace`), the `artifactId` is `azure-pets`, and the version is `1.0.0-beta.1`.
```xml
<dependencies>
  <dependency>
    <groupId>com.azure.pets</groupId>
    <artifactId>azure-pets</artifactId>
    <version>1.0.0-beta.1</version>
  </dependency>
</dependencies>
```

Reply:
thanks so much!